src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -217,6 +233,18 @@ public sealed partial class JsonSerializerOptions
         public bool WriteIndented { get { throw null; } set { } }
         public System.Text.Json.Serialization.JsonConverter GetConverter(System.Type typeToConvert) { throw null; }
     }
+    public partial class JsonString : System.Text.Json.JsonNode, System.IEquatable<System.Text.Json.JsonString>
```

Comment:
Shouldn't this also override the ToString() to do a _value.ToString()? that should probably also be added to the other classes like Bool and Number

Reply:
Comment addressing it: https://github.com/dotnet/corefx/pull/40107#issuecomment-519271307

Reply:
Maybe I can use current solution for `JToken` for now.

Reply:
Whether we decide to use the Json representation or the .NET representation, I think we should still override the ToString method. I would probably pick @JamesNK for now and use the json representation and we can revise that decision during API Review.